src/main/scala/dotty/dokka/tasty/SyntheticSupport.scala

```diff
@@ -70,6 +72,24 @@ trait SyntheticsSupport:
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
     sym.is(dotc.core.Flags.Extension)
   }
+  /* We need there to filter out symbols with certain flagsets, because these symbols come from compiler and TASTY can't handle them well. 
+  They are valdefs that describe case companion objects and cases from enum.
+  TASTY crashed when calling _.tree on them.
```

Comment:
@abgruszecki should we reported it as a bug in TASTY?

Reply:
We get these symbols by hacking into the compiler, so definitely not. OTOH we may extend Tasty with some functionality based on what we needed to hack into in the doctool, but that's basically a separate issue.